Resources/Prototypes/Entities/Clothing/Hands/gloves.yml

```diff
@@ -169,3 +172,16 @@
   - type: Clothing
     sprite: Clothing/Hands/Gloves/fingerless.rsi
 
+- type: entity
+  parent: ClothingHandsBase
+  id: ThievingGloves
+  name: black gloves #Intentionally named after regular gloves, they're meant to be sneaky.
+  description: Regular black gloves that do not keep you from frying.
```

Comment:
I think the description should be diff so someone can tell in-game that they're the thieving gloves upon inspection.